guides/source/association_basics.md

```diff
@@ -2724,3 +2724,102 @@ will run a query like:
 ```sql
 SELECT "vehicles".* FROM "vehicles" WHERE "vehicles"."type" IN ('Car')
 ```
+
+Delegated types
+------------------------------
+
```

Comment:
This starts abruptly, lets add example of what the problem is first?

Reply:
ok. Correct the content 👍

Reply:
@vipulnsward 

Thank you for your review.
After presenting the `STI` problem, changed the explanation to use an example.
How about an explanation after the change?